test/testmachinery/shoot/enable_rotate_certs_disable_test.go

```diff
@@ -0,0 +1,107 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package shoot
+
+import (
+	"context"
+	"time"
+
+	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/gardener/gardener/test/framework"
+	"github.com/gardener/gardener/test/utils/access"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/gardener/gardener-extension-registry-cache/pkg/apis/registry/v1alpha3"
+	"github.com/gardener/gardener-extension-registry-cache/test/common"
+)
+
+const (
+	rotateCertTestTimeout        = 60 * time.Minute
+	rotateCertTestCleanupTimeout = 20 * time.Minute
+)
+
+var _ = Describe("Registry Cache Extension Tests", Label("cache"), func() {
+	f := framework.NewShootFramework(nil)
+
+	f.Disruptive().Beta().CIt("should enable extension, rotate Certificate, disable extension", func(parentCtx context.Context) {
```

Comment:
I am wondering shouldn't this test be serial as well? What it means serial in TM term? It means serial wrt to component tests or serial wrt to all other component test suites? My concern here is to avoid executing this test in parallel with some other component test.

Reply:
It seems that the `Destructive` test runs with no other tests, so it doesn't need to be `Serial` either:
https://github.com/gardener/gardener/blob/52f6464a347fb8e9103ae5d1599dcfb490a5a421/docs/development/testmachinery_tests.md?plain=1#L159C1-L162C156z.